notes/use-cases.md

```diff
@@ -0,0 +1,33 @@
+# Use Cases
+
+1 - Authentication and Authorization
+
+* Alice wants to log into her government services account.
+
+* Alice wants to access the shared class notes in her University group. For this she needs to authenticate, so that she can access the private group resources she has been given explicit access to as part of a group.
+
+2 - Self-description
+
+* Alice wants to have her name and photo displayed in her profile in the different apps she uses, as opposed to having to upload and fill in her info for each individual app.
+
+* Alice wants to connect her alternative profiles to her WebID so that she can select which one to use for different apps. She has a profile for work, with a professional-looking profile picture and her full name, and she also has a profile she keeps private and only uses for dating apps, with only her first name and a different profile picture.
+
+* Alice wants to connect her other social accounts to her WebID profile, so that people interfacing with her Solid Profile know which accounts are legitimate/belong to her.
+
+* Alice is a whistleblower and wants to post information online that, if tied to her real identity, could put her in danger. Her profile then consists of only a pseudonym, so that people interfacing with the information she posts know it all comes from a particular whistleblower, but without exposing her identity.
+
+* Alice wants to host her WebId profile herself on her own server instead of relying on a provider. Her profile is readable by Solid apps, so her friends can follow her activities on social apps, but she updates it manually instead of using an app.
+
+* Alice wants to display proof of her age to a bartender so that she can prove that she can legally drink her beer.
+
+* Alice wants apps to know what her preferred language is, so the the user interface can be displayed to her in the language she's most fluent at.
+
+* Alice wants to list her emergency contact in her WebId Profile, so that in case of emergency people know who to contact.
```

Comment:
```suggestion
* Alice wants to list her emergency contact in her Solid WebId Profile, so that in case of an emergency, people know who to contact.
```